Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -3183,8 +3179,7 @@ inputs:
   playbookInputQuery:
 - key: RemediationNotificationSubject
   value:
-    simple: |-
-      A new security risk was addressed on an external service owned by your team
+    simple: A new security risk was addressed on an external service owned by your team
```

Comment:
```suggestion
    simple: A new security risk was addressed on an external service owned by your team.
```

Reply:
This is an email/ticketing system subject so I don't think it will need a period